<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="PC-NamLe" failures="2" tests="7" name="DemoFirstTest" time="0.167" errors="0" timestamp="2025-06-06T11:11:03 ICT">
  <testcase classname="secondpackage.firstclass" name="FirstMethod" time="0.012"/>
  <testcase classname="secondpackage.firstclass" name="SecondtMethod" time="0.004"/>
  <testcase classname="firstpackage.firstclass" name="FirstMethod" time="0.004"/>
  <testcase classname="secondpackage.secondclass" name="FirstMethod" time="0.009"/>
  <testcase classname="secondpackage.secondclass" name="SecondtMethod" time="0.003"/>
  <testcase classname="firstpackage.secondclass" name="FirstMethod" time="0.028">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at firstpackage.secondclass.FirstMethod(secondclass.java:12)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- FirstMethod -->
  <testcase classname="firstpackage.secondclass" name="SecondtMethod" time="0.007">
    <failure type="java.lang.AssertionError" message="expected [world] but found [hello]">
      <![CDATA[java.lang.AssertionError: expected [world] but found [hello]
at firstpackage.secondclass.SecondtMethod(secondclass.java:20)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- SecondtMethod -->
  <testcase classname="firstpackage.firstclass" name="SecondtMethod" time="0.0">
    <ignored/>
  </testcase> <!-- SecondtMethod -->
</testsuite> <!-- DemoFirstTest -->
